### PR TITLE
Update to Minecraft 1.21.3

### PR DESCRIPTION
**Pull request by JRoy**
~~All changes should be accounted for less items.json. Need to figure out experimental items filtering at runtime.~~

### CODE REVIEW

File: Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java [hunk 1]
```diff
@@ -609,7 +609,7 @@ public void onPlayerBucketEmpty(final PlayerBucketEmptyEvent event) {
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
-        final String cmd = event.getMessage().toLowerCase(Locale.ENGLISH).split(" ")[0].replace("/", "").toLowerCase(Locale.ENGLISH);
+        final String cmd = event.getMessage().split(" ")[0].replace("/", "").toLowerCase(Locale.ENGLISH);
```
Comment (mdcfe) [inline Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java:612]: What does this fix?

File: Essentials/src/main/java/com/earth2me/essentials/MobCompat.java [hunk 2]
```diff
@@ -299,35 +292,4 @@ private Villager.Profession asEnum() {
             return RegistryUtil.valueOf(Villager.Profession.class, newProfession, oldProfession);
         }
     }
-
-    public enum BoatVariant {
```
Comment (mdcfe) [inline Essentials/src/main/java/com/earth2me/essentials/MobCompat.java]: Does this drop support for non-oak boats below 1.21.3?

File: Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java (hunk 1 above)
Comment (JRoy) [reply Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java:612]: nothing but why lower case the string two times

File: Essentials/src/main/java/com/earth2me/essentials/MobCompat.java (hunk 2 above)
Comment (JRoy) [reply Essentials/src/main/java/com/earth2me/essentials/MobCompat.java]: did not test

**Review by mdcfe — APPROVED** [review]

**Pull request merged**